app/views/puzzles/_puzzles_table.html.erb

```diff
@@ -20,6 +23,9 @@
             &nbsp;
           <% end %>
         </td>
+        <% if actions == :archived %>
+          <td><%= time_ago_in_words(puzzle.sent_at) %> ago</td>
```

Comment:
This is fine, I'd rather create a helper method

```ruby
def time_ago(past_time)
  "#{time_ago_in_words(past_time)} ago"
end
```

```suggestion
          <td><%= time_ago(puzzle.sent_at) %></td>
```

Reply:
Done